components/sections/Layout/Header/Header.tsx

```diff
@@ -22,16 +24,37 @@ export const Header = () => {
         </Link>
         <nav className="w-full">
           <ul className="flex justify-end items-center pr-10">
-            {HEADER_NAVIGATION.map(({ name, link }) => {
+            {HEADER_NAVIGATION.map(({ name, link, submenu }) => {
               const activePage = router.asPath.includes(link);
               return (
                 <li
                   key={name}
                   className={`${activePage && 'border-b-2 border-b-primary-yellow'} hover:text-primary-yellow text-white mx-2 `}
                 >
-                  <Link href={link}>
-                    <a>{t(name)}</a>
-                  </Link>
+                  {submenu === undefined ? (
+                    <Link href={link}>
+                      <a>{t(name)}</a>
+                    </Link>
+                  ) : (
+                    <>
+                      <button onClick={() => setShowDropdown(!showDropdown)}>{t(name)}</button>
+                      <div
+                        className={`absolute z-10 ${showDropdown ? '' : 'hidden'} bg-white divide-y divide-gray-100 rounded shadow w-44`}
```

Comment:
Building your own dropdown is 99.9 percent of the time a bad idea. It takes months to get it right. For example, yours is not keyboard navigable and does not close automatically when I click outside of it. Please use a dropdown component from an unstyled UI library. One of these would be best: 

* https://headlessui.dev/react/menu
* https://www.radix-ui.com/docs/primitives/components/dropdown-menu

You can easily style both with Tailwind :)